File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/AppletBox/ApplicationScenario/ExampleApplicationPage.cs
using System;
using System.Text;
using System.Data;
using System.Drawing;
using System.ComponentModel;
using System.Collections.Generic;

using Gizmox.WebGUI.Forms;
using Gizmox.WebGUI.Common;

namespace CompanionKit.Controls.AppletBox.ApplicationScenario
{
    public partial class ExampleApplicationPage : UserControl
    {
        public ExampleApplicationPage()
        {
            InitializeComponent();
        }

        /// <summary>
        /// Handles Load event of the example' UserControl.
        /// </summary>
        private void ExampleApplicationPage_Load(object sender, EventArgs e)
        {
            // Add parameters for DAEJA Applet.
            this.demoAppletBox.AddParameter("cabbase", "viewone.cab");
            this.demoAppletBox.AddParameter("demo", "document");
            this.demoAppletBox.AddParameter("backcolor", "192,192,192");
            this.demoAppletBox.AddParameter("activecolor", "154,156,206");
            this.demoAppletBox.AddParameter("flipOptions", "true");
            this.demoAppletBox.AddParameter("scale", "ftow");
            this.demoAppletBox.AddParameter("viewmode", "thumbsleft");
            this.demoAppletBox.AddParameter("pagekeys", "true");
            this.demoAppletBox.AddParameter("defaultprintdoc", "true");
            this.demoAppletBox.AddParameter("externalMagnifier", "false");
            this.demoAppletBox.AddParameter("page1", "../docs/mixed/p1.tif");
            this.demoAppletBox.AddParameter("page2", "../docs/mixed/p2.jpg");
            this.demoAppletBox.AddParameter("page3", "../docs/mixed/p3.tif");
            this.demoAppletBox.AddParameter("page4", "../docs/mixed/p4.jpg");
            this.demoAppletBox.AddParameter("thumb1", "../docs/mixed/p1-t.tif");
            this.demoAppletBox.AddParameter("thumb2", "../docs/mixed/p2-t.jpg");
            this.demoAppletBox.AddParameter("thumb3", "../docs/mixed/p3-t.tif");
            this.demoAppletBox.AddParameter("thumb4", "../docs/mixed/p4-t.jpg");
            this.demoAppletBox.AddParameter("thumbLabel1", "Features");
            this.demoAppletBox.AddParameter("thumbLabel2", "Fruit");
            this.demoAppletBox.AddParameter("thumbLabel3", "High resolution");
            this.demoAppletBox.AddParameter("thumbLabel4", "Daeja Image");
            this.demoAppletBox.AddParameter("enhance", "true");
            this.demoAppletBox.AddParameter("enhancemode", "1");
            this.demoAppletBox.AddParameter("allowSelectAnnotation", "true");
            this.demoAppletBox.AddParameter("version3Features", "true");
            this.demoAppletBox.AddParameter("annotationEncoding", "UTF8");
            this.demoAppletBox.AddParameter("annotate", "true");
            this.demoAppletBox.AddParameter("annotateEdit", "true");
        }
    }
}

File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/AppletBox/ApplicationScenario/ExampleApplicationPage.vb
Namespace CompanionKit.Controls.AppletBox.ApplicationScenario

	Public Class ExampleApplicationPage

		Public Sub New()

			' This call is required by the Visual WebGui UserControl Designer.
			InitializeComponent()

	        

		End Sub

        Private Sub ExampleApplicationPage_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load
            Me.demoAppletBox.AddParameter("cabbase", "viewone.cab")
            Me.demoAppletBox.AddParameter("demo", "document")
            Me.demoAppletBox.AddParameter("backcolor", "192,192,192")
            Me.demoAppletBox.AddParameter("activecolor", "154,156,206")
            Me.demoAppletBox.AddParameter("flipOptions", "true")
            Me.demoAppletBox.AddParameter("scale", "ftow")
            Me.demoAppletBox.AddParameter("viewmode", "thumbsleft")
            Me.demoAppletBox.AddParameter("pagekeys", "true")
            Me.demoAppletBox.AddParameter("defaultprintdoc", "true")
            Me.demoAppletBox.AddParameter("externalMagnifier", "false")
            Me.demoAppletBox.AddParameter("page1", "../docs/mixed/p1.tif")
            Me.demoAppletBox.AddParameter("page2", "../docs/mixed/p2.jpg")
            Me.demoAppletBox.AddParameter("page3", "../docs/mixed/p3.tif")
            Me.demoAppletBox.AddParameter("page4", "../docs/mixed/p4.jpg")
            Me.demoAppletBox.AddParameter("thumb1", "../docs/mixed/p1-t.tif")
            Me.demoAppletBox.AddParameter("thumb2", "../docs/mixed/p2-t.jpg")
            Me.demoAppletBox.AddParameter("thumb3", "../docs/mixed/p3-t.tif")
            Me.demoAppletBox.AddParameter("thumb4", "../docs/mixed/p4-t.jpg")
            Me.demoAppletBox.AddParameter("thumbLabel1", "Features")
            Me.demoAppletBox.AddParameter("thumbLabel2", "Fruit")
            Me.demoAppletBox.AddParameter("thumbLabel3", "High resolution")
            Me.demoAppletBox.AddParameter("thumbLabel4", "Daeja Image")
            Me.demoAppletBox.AddParameter("enhance", "true")
            Me.demoAppletBox.AddParameter("enhancemode", "1")
            Me.demoAppletBox.AddParameter("allowSelectAnnotation", "true")
            Me.demoAppletBox.AddParameter("version3Features", "true")
            Me.demoAppletBox.AddParameter("annotationEncoding", "UTF8")
            Me.demoAppletBox.AddParameter("annotate", "true")
            Me.demoAppletBox.AddParameter("annotateEdit", "true")
        End Sub
    End Class

End Namespace